bot/src/com/mygdx/game/client/bot/BotClient.java

```diff
@@ -1,4 +1,5 @@
 package com.mygdx.game.client.bot;
```

Comment:
@Szomeszp taka porada jak piszesz nowe klasy w Javie i kończysz pracę to użyj skróto Ctrl + Alt + L, robi Ci autorefactor, czyli to co tu Kacper poprawiał z odstępami itd.